dkc/core/rest/file.py

```diff
@@ -91,4 +93,6 @@ def perform_create(self, serializer: FileSerializer):
     @action(detail=True)
     def download(self, request, pk=None):
         file = get_object_or_404(File, pk=pk)
-        return HttpResponseRedirect(file.blob.url)
+        if file.blob:  # FileFields are falsy when not populated with a file
```

Comment:
```suggestion
        if file.blob:  # FieldFile are falsy when not populated with a file
```

The actual instantiated field value is a `FieldFile`: https://docs.djangoproject.com/en/3.1/ref/models/fields/#filefield-and-fieldfile